docs/Manifest Parameters.md

```diff
@@ -722,90 +722,329 @@ The POTA manifest is used to perform both a FOTA and SOTA update at the same tim
-| `<?xml version='1.0' encoding='utf-8'?>` |                        |         R         |                               |
-| `<manifest>`                             | `<manifest>`           |         R         ||
-| `<type><type>`                           | `<type>cmd</type>`     |         R         | will always be 'cmd'          |
-| `<cmd></cmd>`                            | `<cmd>query</cmd>`     |         R         | will always be 'query'        |
-| `<query>`                                | `<query>`              |         R         |                               |
-| `<option></option>`                      | `<option>all</option>` |         R         | [Available Options](Query.md) |
-| `</query>`                               | `</query>`             |         R         |                               |
-| `</manifest>`                            | `</manifest>`          |         R         |                               |
+| XML Tags                                 | Definition                     | Required/Optional | Notes                         |
+|:-----------------------------------------|:-------------------------------|:-----------------:|:------------------------------|
+| `<?xml version='1.0' encoding='utf-8'?>` |                                |         R         |                               |
+| `<manifest>`                             | `<manifest>`                   |         R         |                               |
+| `<type><type>`                           | `<type>cmd</type>`             |         R         | will always be 'cmd'          |
+| `<cmd></cmd>`                            | `<cmd>query</cmd>`             |         R         | will always be 'query'        |
+| `<query>`                                | `<query>`                      |         R         |                               |
+| `<option></option>`                      | `<option>all</option>`         |         R         | [Available Options](Query.md) |
+| `<targetType></targetType>`              | `<targetType>node</targetType>`|         O         | [vision, node] Used when updating either the host of vision cards (host) or vision cards (node) |
+| `<targets></targets>`                    | `<targets><target>389C0A</target></targets>` |         O         | Used when targetType=node.  Designates the Ids of the nodes to update    
+| `</query>`                               | `</query>`                     |         R         |                               |
+| `</manifest>`                            | `</manifest>`                  |         R         |                               |
+
+
+## Query types supported on Edge and HDDL platforms
+  
+| Query Type Supported on Edge only  | Query Type Supported on HDDL only | Query Type Supported on both Edge and HDDL |
+|:-----------------------------------|:----------------------------------|:------------------------------------------:|
```

Comment:
I think the table to look better if 'Query Type' is removed from each column so that it doesn't wrap the text.  The title already tells us that it's Query Type so it should be alright to remove that.